src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +173,16 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_ops(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    std::vector<std::shared_ptr<ov::Node>> ops = _model->get_ops();
+
+    Napi::Array result = Napi::Array::New(env, ops.size());
+    for (size_t i = 0; i < ops.size(); ++i) {
```

Comment:
```suggestion
    for (uint32_t i = 0; i < ops.size(); ++i) {
```
`size_t` may cause errors/warnings on some systems